Giovaniavs/MyProject-Client
# Develop/timeline

## ***What I Did:***
- Created the timeline component.

## ***How to Test:***
1. Run this current branch, click in the **Experiences** button in the navbar and see the section working..
